app/service/auth_svc.py

```diff
@@ -1,19 +1,23 @@
 import base64
+import logging
 from collections import namedtuple
 
 from aiohttp import web, web_request
 from aiohttp.web_exceptions import HTTPUnauthorized, HTTPForbidden
+from aiohttp_jinja2 import render_template
```

Comment:
This made me feel a bit uneasy but I don't have a clear concern other than it potentially blurring lines/boundaries. I.e., the auth service knows about templates and rendering them. Hm.

Reply:
I'm splitting of the DefaultLoginHandler into a separate module, and that `render_template` import will get moved over there